ext/srt/gstsrtobject.c

```diff
@@ -583,6 +591,19 @@ gst_srt_object_install_properties_helper (GObjectClass * gobject_class)
           "Stream ID for the SRT access control", "",
           G_PARAM_READWRITE | GST_PARAM_MUTABLE_READY |
           G_PARAM_STATIC_STRINGS));
+
+  /**
+   * GstSRTSink:authentication:
+   *
+   * Boolean to block streaming until a client connects.  If TRUE,
+   * `srtsink' will stream only when a client is connected.
```

Comment:
Forgotten copy-paste from `GstSRTSink:wait-for-connection`

Reply:
Corrected the comment.